src/Ckeditor.vue

```diff
@@ -82,7 +82,7 @@ export default {
     update (val) {
       let html = this.instance.getData()
       if (html !== val) {
-        this.instance.setData(val)
+        this.instance.setData(val, { internal: true })
```

Comment:
See options for `setData` in the [docs](https://docs.ckeditor.com/ckeditor4/latest/api/CKEDITOR_editor.html#setData)